webapp/src/modules/account/sagas.spec.ts

```diff
@@ -129,3 +146,176 @@ describe('when handling the request to fetch account metrics', () => {
     })
   })
 })
+
+describe('when handling the request to fetch creators accounts', () => {
+  let search: string
+  let accounts: Account[]
+  let addresses: string[]
+
+  describe('when a call to the nftAPI fails', () => {
+    const error = 'request failed with error'
+    let filters: NFTsFetchParams
+
+    describe('when the call with search with a term', () => {
+      beforeEach(() => {
+        search = 'a term'
+        filters = {
+          category: NFTCategory.ENS,
+          search,
+          first: 20,
+          skip: 0
+        }
+      })
+      it('should signal that the request has failed with the request error', () => {
+        return expectSaga(accountSaga, catalystClient)
+          .provide([
+            [
+              call([nftAPI, nftAPI.fetch], filters),
+              Promise.reject(new Error(error))
+            ]
+          ])
+          .put(fetchCreatorsAccountFailure(search, error))
+          .dispatch(fetchCreatorsAccountRequest(search))
+          .silentRun()
+      })
+    })
+  })
+
+  describe('when a call to the accountAPI fails', () => {
```

Comment:
What if we fuse the two describes into one?
```suggestion
  describe('when the request without a search term fails while fetching the Accounts API', () => {
```